sql/rooms/up.sql

```diff
@@ -1,10 +1,93 @@
+CREATE TYPE FloorTypeEnum AS ENUM (
+    'Flat', 
+    'Tiered', 
+    'Other'
+);
+
+CREATE TYPE SeatingTypeEnum AS ENUM (
```

Comment:
Make sure to update `down.sql` to drop the new types

Reply:
done ^^